src/lxc/cgroups/cgfsng.c

```diff
@@ -1284,7 +1284,7 @@ static int rmdir_wrapper(void *data)
 		SYSERROR("Failed to setgid to 0");
 	if (setresuid(nsuid, nsuid, nsuid) < 0)
 		SYSERROR("Failed to setuid to 0");
-	if (setgroups(0, NULL) < 0)
+	if (setgroups(0, NULL) < 0 && errno != EPERM)
```

Comment:
So we'll ignore failures to write due to eperm.  Is that ok?

Reply:
It should only ever mean that there's a "deny" in /proc/[pid]/setgroups but I don't want to open this and read it every time. But we could.